luup files/L_RGBController1.lua

```diff
@@ -1037,6 +1037,49 @@ function doColorTransition(lul_device)
 	end
 end
 
+-- MySensor RGB(W)
+RGBDeviceTypes["MYS-RGBW"] = {
+	getParameters = function (lul_device)
+		return {
+			name = "MySensors RGBW",
+			settings = {
+				{ variable = "ArduinoId", name = "Arduino plugin Id", type = "string" },
+				{ variable = "RadioId",   name = "RGB Node altid",    type = "string" }
+			}
+		}
+	end,
+
+	getColorChannelNames = function (lul_device)
+		return {"red", "green", "blue", "warmWhite", "coolWhite"}
```

Comment:
Hi vosmont.

Thanks for your response.

You are right somehow i did not copy the complete code for the MySensors
support. I once tried it again with a new pull request.

The installation process is indeed a bit weird, this is due to the
combination of Vera and the MySensors module.
The MySensors module creates a RGB controller tile for each RGB device
which is detected, these tiles (Vera devices) are created by Vera as child
devices of the MySensors device.
And Vera only accept child devices being controlled through the master
device.

So the automatically created child device needs to be removed and a new
manual parent device needs to be created and configured. At the MySensors
forum i'm discussing a more convenient install procedure.

Hope this explains the issue a bit.

Greetz,
   Bart a.k.a. Bekkie


On Mon, Oct 12, 2015 at 11:04 AM, vosmont <notifications@github.com> wrote:

> In luup files/L_RGBController1.lua
> <https://github.com/vosmont/Vera-Plugin-RGBController/pull/1#discussion_r41736305>
> :
>
> > @@ -1037,6 +1037,49 @@ function doColorTransition(lul_device)
> >  	end
> >  end
> >
> > +-- MySensor RGB(W)
> > +RGBDeviceTypes["MYS-RGBW"] = {
> > +	getParameters = function (lul_device)
> > +		return {
> > +			name = "MySensors RGBW",
> > +			settings = {
> > +				{ variable = "ArduinoId", name = "Arduino plugin Id", type = "string" },
> > +				{ variable = "RadioId",   name = "RGB Node altid",    type = "string" }
> > +			}
> > +		}
> > +	end,
> > +
> > +	getColorChannelNames = function (lul_device)
> > +		return {"red", "green", "blue", "warmWhite", "coolWhite"}
>
> Hello Bekkie,
> I am not used to the pull requests yet.
> It seems that you have not commited all the code (it lacks "setStatus" and
> "setColor" functions).
>
> Besides, I'm not sure to understand your installation process. You don't
> have to delete the RGB color wheel created by Arduino plugin (which I don't
> know). I think the RGB controller can work with these devices.
> And to create another RGB Controller, you just have to go in "Apps" panel,
> you will find a button "create another".
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vosmont/Vera-Plugin-RGBController/pull/1/files#r41736305>
> .
>